code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -217,39 +249,38 @@
 		if(!process_scan(target))
 			target = null
 
-	if(!target && bot_cover_flags & BOT_COVER_EMAGGED) // When emagged, target humans who slipped on the water and melt their faces off
-		target = scan(/mob/living/carbon)
+	if(!target)
+		var/list/scan_targets = list(
+			/obj/effect/decal/cleanable,
+			/obj/effect/decal/remains,
+		)
 
-	if(!target && pests) //Search for pests to exterminate first.
-		target = scan(/mob/living/simple_animal)
+		if(bot_cover_flags & BOT_COVER_EMAGGED) // When emagged, target humans who slipped on the water and melt their faces off
+			scan_targets += list(/mob/living/carbon)
+		if(pests)
+			scan_targets += list(/mob/living/simple_animal)
+		if(trash)
+			scan_targets += list(
+				/obj/item/trash,
+				/obj/item/food/deadmouse,
+			)
```

Comment:
Doesn't this change behavior? 

In the past it would scan for carbons, then simple animals, then cleanables and trash and stuff.
Now, it scans for cleanables, then carbons.

Reply:
It is, I’ll add it to the changelog

Reply:
Actually I’ll change it again later, I’ll draft this for now